packages/cashier/src/modules/crypto-transactions/components/crypto-transactions-side-note-resent-transaction/crypto-transactions-side-note-recent-transaction.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useCallback } from 'react';
+import { Button, Loading, SideNote, Text } from '@deriv/components';
+import { useCryptoTransactions, useCurrentCurrencyConfig } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+import { Localize, localize } from '@deriv/translations';
+import classNames from 'classnames';
+import { useCashierStore } from '../../../../stores/useCashierStores';
+import { cryptoTransactionMapper } from '../../helpers';
+import './crypto-transactions-side-note-recent-transaction.scss';
+
+const CryptoTransactionsSideNoteRecentTransaction: React.FC = observer(() => {
```

Comment:
NIT. Maybe?
```suggestion
const CryptoRecentTransactionSideNote: React.FC = observer(() => {
```

Reply:
@nijil-deriv I agree the name is long 😅

What I'm following now is `ModuleNameComponentTypeComponentName`, So we can have better auto-complete, For example by typing `DepositFiat` we will get auto-complete for everything under the `DepositFiat` module.

Just trying out tho, Once things are separated we can rename things easily 🙇🏻

Reply:
Good point. In that case, would `ModuleNameComponentNameComponentType` make more sense? Guess it's more of a personal preference. For example, `CryptoTransactionsRecentTransactionSideNote`

Reply:
@nijil-deriv Makes sense, We can discuss this in our meeting to get input from everyone and come up with a pattern and then make a PR to just rename things 🤔